pkg/controller/v1beta1/inferenceservice/utils/utils.go

```diff
@@ -64,6 +64,12 @@ func IsMMSPredictor(predictor *v1beta1api.PredictorSpec) bool {
 		}
 		return false
 	} else {
+		if predictor.Model != nil {
+			// we do not support model agent for Huggingface runtime
+			if predictor.Model.ModelFormat.Name == "huggingface" {
+				return false
+			}
+		}
```

Comment:
Oops you are faster. I am fixing this in https://github.com/kserve/kserve/pull/3449 as well. Would you like to merge that first and then rebase?

Reply:
sounds good

Reply:
My PR has a couple of other issues